### DISCUSSION TITLE
Can the add on send data to two home assistant instances?

### DISCUSSION THREAD

**Discussion by m90att** [kellerza/sunsynk#297] [Q&A]
I'm migrating HA from HA Yellow to an HP Elitedesk (proxmox) but it's going to be a long process as I'm refining it as I go.  I have the add on sending data to my existing HA but I also have an MQTT broker running in proxmox, supplying the new HA with other MQTT data.  I'm wondering if there's a way to get my solar data to both HA instances. Does anyone know?

**Comment by kellerza** [comment]
This is something that you can solve using an MQTT bridge. 

Mosquitto config (i.e. bridge.conf) applied to ONE of the Mosquitto instances

```
# client name
connection mqtt01
# remote instance
address 192.168.1.10:1883

# Sunsynk sensor data
topic SUNSYNK/status/# both 0

# HA auto-discovery info (replace SERIAL with your inverter serial #)
topic homeassistant/sensor/SERIAL/# both 0
topic homeassistant/binary_sensor/SERIAL/# both 0
topic homeassistant/switch/SERIAL/# both 0
topic homeassistant/select/SERIAL/# both 0
topic homeassistant/number/SERIAL/# both 0
```
